aya/src/symbols.rs

```diff
@@ -0,0 +1,688 @@
+//! Utilities for resolving symbols from addresses obtained via stack traces
+use std::{
+    cell::RefCell,
+    collections::BTreeMap,
+    fs::File,
+    io::{self, BufRead, BufReader},
+    path::{Path, PathBuf},
+};
+
+use addr2line::{
```

Comment:
As you've noticed in the USDT PR, we use the `object` crate to parse ELF and I think we should use it here too

Reply:
Given `addr2line` is a tiny wrapper around `object`, what would be the win? It seems like it would just duplicate the functionality. Plus at least I don't have the domain knowledge about ELF files at the moment and time to dig into it, so if that's preferable someone else would need to look into it.

Reply:
Oh, I did not realize that :) It seems that it can do demangling too, and that's behind cargo features. I think we should default to having those features off so that we don't bring in more dependencies for people that are not going to implement profilers.

In any case we shouldn't re-export any symbols from `addr2line` in the same way that we don't re-export anything from any dependencies.